### ISSUE TITLE
Angle minimisation Firefox problem

### ISSUE THREAD

**Issue by jrencz** [css/csso#229]
Firefox des not seem to like `0` as angle in `linear-gradient`:

```
linear-gradient(0deg, #2B2B2B 1px, transparent 1px),  // valid
// After csso optimisation. 
linear-gradient(0, #2B2B2B 1px, transparent 1px); // invalid but treated as valid in Chrome and Safari. Not in Firefox.
```

It seems to angle has to have any unit.
Quick fix is to pass `360deg` instead of `0deg` but it's just a hack.

**Comment by Wizza** [comment]
#234 will fix this issue.

**Comment by lahmatiy** [comment]
Fixed in 1.3.12
